src/components/UI/homepage/HomeHero.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Button, Flex, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+import NextLink from 'next/link';
+
+import { DOCS_PAGE, DOWNLOADS_PAGE } from '../../../constants';
+
+export const HomeHero: FC = () => {
+  return (
+    <Stack border='2px solid' borderColor='brand.light.primary' px={4} py={{ base: 8, md: 5 }}>
+      <Box mb={4}>
+        <Box
+          as='h1'
+          textStyle='h1'
+          mb={{ base: 2, md: 4 }}
+          textAlign={{ base: 'center', md: 'left' }}
+        >
+          go-ethereum
+        </Box>
+
+        <Text textStyle='homepage-description'>
+          Official Go implementation of the Ethereum protocol
+        </Text>
+      </Box>
+
+      <Flex
+        direction={{ base: 'column', md: 'row' }}
+        alignItems={{ base: 'center', md: 'flex-start' }}
+      >
+        <Flex direction='column' alignItems='center' mr={{ md: 6 }}>
```

Comment:
These buttons dont match the design. Not sure if this was changed, if yes all good, just calling it out. 

Design system:
<img width="149" alt="Screen Shot 2022-10-09 at 9 22 19 AM" src="https://user-images.githubusercontent.com/15589226/194765166-cb3a79e7-8a4c-49c3-8e64-3a5e33fd110c.png">

Implementation:
<img width="438" alt="Screen Shot 2022-10-09 at 9 24 06 AM" src="https://user-images.githubusercontent.com/15589226/194765178-99cf66de-98be-408e-8a4d-a902d8efe538.png">

For what its worth I think this looks good still.